### ISSUE TITLE
Show compilation error message in asm view

### ISSUE THREAD

**Issue by mattgodbolt** [compiler-explorer/compiler-explorer#189]
Some some cases (e.g. `static_assert` coming from an include), the error appears in the compiler output panel, but in the code panel there's only the note, not the error.

https://godbolt.org/g/DClrTX

It would be nice to put something maybe at the include location (also for recursive includes)

**Comment by mattgodbolt** [comment]
Perhaps in general we should show the compiler error output if there's a compilation failure instead of the generically unhelpful `<compilation error>`.

**Comment by AndrewPardoe** [comment]
Matt, is this still needed now that we have the Output button?

I've made a change for diagnostics to go to the asm window. While investigating why we always turn on `-fdiagnostics-color`, I discovered you've added the Output button (where color is quite useful.) It seems that showing diagnostic output in the asm window isn't necessary anymore. 

Maybe the best change would be to the string `<Compilation failed>`. Adding another line that says `Compiler output available in Output window` would help in discovery of the output window.
